docs/source/using-discovery/usingslurm.rst

```diff
@@ -3,51 +3,95 @@
 ***********
 Using Slurm
 ***********
+.. contents:: Table of Contents
+   :depth: 4
 
-Slurm (Simple Linux Utility Resource Management) is the software on Discovery
-that lets you do the following:
+Overview
+========
+Slurm (Simple Linux Utility for Resource Management) is an open-source job scheduler and resource manager developed at Lawrence Livermore National Laboratory and is widely-used in high-performance computing (HPC) environments, including academic and research institutions, government labs, and private industry. Slurm is designed to efficiently allocate and reasonably manage resources (e.g., CPUs, memory, GPUs) among multiple users and jobs, and is known for its scalability, reliability, and flexibility.
 
-* view information about the cluster
-* monitor your jobs
-* schedule your jobs on Discovery
-* view information about your account
+Slurm provides features to support the efficient management of HPC resources, including job submission, scheduling, prioritization, and accounting. For example, Slurm allows users to submit jobs and request resources and manages the allocation and distribution of resources across the cluster. Slurm can also handle job dependencies, allowing users to define dependencies between jobs and specify when specific jobs should start.
 
-:ref:`using_srun` and :ref:`using_sbatch` provide you with a few examples to help get you familiar
-with Slurm and be able to submit basic jobs on Discovery.
+One of the key features of Slurm is its ability to support heterogeneous clusters, where different compute nodes have varying hardware specifications. Slurm can manage resources on various hardware and software configurations. It provides a way to specify the job requirements to ensure scheduling with the appropriate nodes.
+
+Another critical feature of Slurm is its ability to handle complex workflows and dependencies. For example, users can specify complex workflows of jobs where specific jobs depend on others, and Slurm will manage the scheduling and execution of these workflows.
+
+Overall, Slurm is a powerful and versatile tool for managing HPC resources. Its flexibility, scalability, and reliability make it an essential tool for many HPC environments.
+
+Slurm on Discovery
```

Comment:
I think it would be helpful to combine this with the previous section, the Slurm on Discovery description isn't very thorough, so it might be best to just eliminate the first paragraph and adding a bit about Discovery using Slurm. I can work on a draft and send it to you if you would like.

Reply:
Yes. That would be nice if you can combine (or I can, just not entirely clear on the specifics on what you have in mind)

Reply:
Honestly, I couldn't remember exactly what I was thinking, but I did make an update to try and make it feel more Discovery-focused, just by moving the Slurm on Discovery heading up a few paragraphs.